api/XDS_PROTOCOL.md

```diff
@@ -245,7 +246,8 @@ In general, to avoid traffic drop, sequencing of updates should follow a
 * CDS updates (if any) must always be pushed first.
 * EDS updates (if any) must arrive after CDS updates for the respective clusters.
 * LDS updates must arrive after corresponding CDS/EDS updates.
-* RDS updates related to the newly added listeners must arrive in the end.
+* RDS updates related to the newly added listeners must arrive in the end, but before VHDS updates
```

Comment:
Is it better to say

RDS updates related to the newly added listeners must arrive after CDS/EDS/LDS updates
VHDS updates (if any) related to the newly added Route Configurations must arrive after RDS updates

Reply:
That sounds better I think.

Reply:
Updated.